### ISSUE TITLE
Error on missing usage-file

### ISSUE THREAD

**Issue by alikhajeh1** [infracost/infracost#1574]
With v0.9.22 of the CLI, I took me a while to debug an issue as `infracost breakdown --path examples/terraform --usage-file BLAH.yaml` worked even though the file BLAH.yaml didn't exist. We should throw a nice error like we do for the config-file:
```
> infracost breakdown --config-file BLAH.yaml
Error: config file does not exist at BLAH.yaml
```

**Comment by alikhajeh1** [comment]
@vdmgolub correct, the error should only be if: `--usage-file` is used AND the file does not exist AND `--sync-usage-file` is not being used.

**Comment by aliscott** [comment]
Closing this since it is stale.
